phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/ViewEngines.java

```diff
@@ -68,6 +68,7 @@ public void changed(ObservableValue<? extends Boolean> observableValue, Boolean
     private final CombinedEnginesFilter currentFilter = new CombinedEnginesFilter();
     private Consumer<CombinedEnginesFilter> onApplyFilter = (filter) -> {};
     private Consumer<EngineCategoryDTO> onSelectCategory;
+    private Consumer<List<EngineCategoryDTO>> onSetDefaultEngineIcons;
```

Comment:
When will this consumer be triggered? What does it do? I would wish for some JavaDoc for all used consumers.